angular-workspace/projects/ni/nimble-angular/src/directives/patterns/link/nimble-link-base.ts

```diff
@@ -0,0 +1,66 @@
+import { ElementRef, Injectable, Input, Renderer2 } from '@angular/core';
+import type { AnchorBase } from '@ni/nimble-components/dist/esm/anchor-base';
+
+/**
+ * Base class for directives providing link attributes.
+ */
+@Injectable()
+export class NimbleLinkBase<T extends AnchorBase> {
```

Comment:
Should match the base class style of icon-base. Class name should be NimbleAnchorBaseDirective and file should be 
/directives/anchor-base/nimble-anchor-base.directive.ts (aligns with the /directives/icon-base conventions).

Reply:
Done